pkg/oc/admin/diagnostics/diagnostics/cluster/scc.go

```diff
@@ -0,0 +1,95 @@
+package cluster
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	kerrs "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kubernetes/pkg/apis/authorization"
+	authorizationtypedclient "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset/typed/authorization/internalversion"
+
+	"github.com/openshift/origin/pkg/cmd/server/bootstrappolicy"
+	"github.com/openshift/origin/pkg/oc/admin/diagnostics/diagnostics/types"
+	"github.com/openshift/origin/pkg/oc/admin/diagnostics/diagnostics/util"
+	policycmd "github.com/openshift/origin/pkg/oc/admin/policy"
+	securityapi "github.com/openshift/origin/pkg/security/apis/security"
+	securitytypedclient "github.com/openshift/origin/pkg/security/generated/internalclientset/typed/security/internalversion"
+)
+
+// ClusterSCC is a Diagnostic to check that the default securitycontextconstraints match expectations
+type ClusterSCC struct {
+	SCCClient securitytypedclient.SecurityContextConstraintsInterface
+	SARClient authorizationtypedclient.SelfSubjectAccessReviewsGetter
+}
+
+const SccName = "ClusterSCC"
```

Comment:
Why not SCC?

Reply:
I standarized the name under `pkg/oc/admin/diagnostics/diagnostics/cluster` with `ClusterXXX`, after reviewed other diagnostics.

(e.g)
https://github.com/openshift/origin/blob/ea90a090da768e9a505b0ca2368927d7d25210f1/pkg/oc/admin/diagnostics/diagnostics/cluster/registry.go#L31

https://github.com/openshift/origin/blob/ea90a090da768e9a505b0ca2368927d7d25210f1/pkg/oc/admin/diagnostics/diagnostics/cluster/router.go#L32


But it looks like `ClusterXXX` was not necessary(?) I realized that some did not have it.

https://github.com/openshift/origin/blob/ea90a090da768e9a505b0ca2368927d7d25210f1/pkg/oc/admin/diagnostics/diagnostics/cluster/service_externalip.go#L28

Reply:
I think that `ClusterSCC` shoud be used here. In case someone wants to add diagnostics related to `scc` in the future, (if I used `SCC` here), `SCC` will include wide meaning and may cause confusion.

Reply:
Then the correct approach would be to have a top level scc check with this as a child.  ClusterSCC makes no sense because SCC has not namespace equivalent like RBAC's Cluster/Role.

Reply:
> ClusterSCC makes no sense because SCC has not namespace equivalent like RBAC's Cluster/Role.

Since SCC is effected by SA, SCC has a relation to namespace and sometimes need namespace based check. Actually, logging namespace has its SCC diagnostics as: https://github.com/openshift/origin/blob/8ac6446f8e4f2db4dea4448b36de7af69ee6860b/pkg/oc/admin/diagnostics/diagnostics/cluster/aggregated_logging/scc.go

Reply:
Sure, but the SCC check itself is always cluster scoped.  It does not matter that the user bound to it exists only in a namespace.

Reply:
+1 for renaming it to be "SCC".

Reply:
I understand. Updated to "SCC"